app/controllers/items_controller.rb

```diff
@@ -1,15 +1,18 @@
-class ItemsController < ApplicationController
+class ItemsController < OauthController
```

Comment:
This update causes us to lose the inheritance from `application.html.erb`. That's why we have no styles and the outer tags of DOMs. I am not sure how to fix it yet.

Reply:
It is fixed now by not using `initialize` method in `OauthController`. Thanks to @nodanaonlyzuul